homeassistant/components/refoss/const.py

```diff
@@ -19,3 +29,65 @@
 COORDINATOR = "coordinator"
 
 MAX_ERRORS = 2
+
+ENERGY = "energy"
+ENERGY_RETURNED = "energy_returned"
+
+
+@dataclass
+class UnitOfMeasurement:
+    """Describes a unit of measurement."""
+
+    unit: str
+    device_classes: set[str]
+    aliases: set[str] = field(default_factory=set)
+    conversion_unit: str | None = None
+    conversion_fn: Callable[[float], str] | None = None
+
+
+UNITS = (
```

Comment:
Why are you doing manual unit conversions if Home Assistant has support for it built in?